doc/rho.1

```diff
@@ -0,0 +1,408 @@
+.TH rho 1 "August 10, 2017" "version 0.1" "rho User Commands"
+.SH NAME
+rho \- Easily discover and manage systems on your network.
+
+.SH SYNOPSIS
+rho
+.I command subcommand [options]
+
+.SH DESCRIPTION
+rho is a network discovery tool to identify the number of systems running on a network, their operating system, hardware, and release numbers for almost any Linux or Unix flavor. Being able to identify the systems running on the network is a vital component to managing licenses and renewals. Ultimately, discovery is part of the larger sysadmin task of managing inventories.
```

Comment:
I realize this is imported from Candlepin, but how about changing "their operating system, hardware, and release numbers" to "their operating system and versions of some key packages" or something similar? I don't think the current text is a good description of Rho